Gem/Code/enabled_gems.cmake

```diff
@@ -66,4 +66,6 @@ set(ENABLED_GEMS
     Terrain
     Profiler
     Stars
+    AWSGameLift
```

Comment:
Lets discuss if we have these on by default

Reply:
Pretty sure it needs to be off by default. I just accidently checked this in. Reverted!